common/plugins/org.jboss.tools.common.el.core/src/org/jboss/tools/common/el/core/ca/AbstractELCompletionEngine.java

```diff
@@ -161,8 +166,11 @@ public IRelevanceCheck createRelevanceCheck(IJavaElement element) {
 	 * @see org.jboss.tools.common.el.core.resolver.ELResolver#resolve(org.jboss.tools.common.el.core.resolver.ELContext, org.jboss.tools.common.el.core.model.ELExpression)
 	 */
 	public ELResolution resolve(ELContext context, ELExpression operand, int offset) {
-		List<Var> vars = new ArrayList<Var>();
+		List<Var> vars = EMPTY_VARS;
 		Var[] array = context.getVars(offset);
+		if(array.length > 0) {

```

Comment:
Yes, it may be ok in this case. Arrays.asList should be used carefully, as it returns fixed size list - attempts to add/remove an element will result in exception.